docs/reference/mapping/types/nested.asciidoc

```diff
@@ -189,7 +189,7 @@ The following parameters are accepted by `nested` fields:
 
 Because nested documents are indexed as separate documents, they can only be
 accessed  within the scope of the `nested` query, the
-`nested`/`reverse_nested`, or <<nested-inner-hits,nested inner hits>>.
+`nested`/`reverse_nested` aggregations, or <<nested-inner-hits,nested inner hits>>.
```

Comment:
nit: I'd prefer the singular `aggregation` here.

Reply:
I prefer it in plural to better state that those are 2 different aggregations.